index.js

```diff
@@ -30,15 +30,14 @@
 
 'use strict';
 
-var Handlebars = require('./lib/vendors/handlebars'),
-	TemplateProvider = require('./lib/TemplateProvider');
+const Handlebars = require('./lib/vendors/handlebars');
+const TemplateProvider = require('./lib/TemplateProvider');
 
 module.exports = {
-	register: function (locator, config) {
-		config = config || {};
+	register(locator, config) {
 		locator.registerInstance('handlebars', Handlebars);
 		locator.register('templateProvider', TemplateProvider, config, true);
```

Comment:
Service locator does not have a `config` parameter anymore, the third parameter is `isSingleton`.